src/handler_forward.js

```diff
@@ -154,6 +154,10 @@ export default class HandlerForward extends HandlerBase {
 
         this.srcResponse.writeHead(response.statusCode, headers);
         response.pipe(this.srcResponse);
+
+        response.once('end', () => {
```

Comment:
Comment here too perhaps?

Reply:
is there any situation where this event listener is attached more than once and never triggered, since the `end` event is [notoriously flakey](https://nodejs.org/api/stream.html#stream_event_end)? might lead to the MaxListenersExceededWarning

Reply:
@pocesar If this isn't called then `error` will which will happen to destroy the socket.